example/Cargo.toml

```diff
@@ -8,8 +8,7 @@ edition = "2021"
 [workspace]
 
 [dependencies]
-#momento = "0.15.0"
-momento = { path = "../" }
-tokio = { version = "1.18.2", features = ["full"] }
-uuid = { version = "1.2.2", features = ["v4"] }
 futures = "0.3.30"
+momento = { path = "../" }
```

Comment:
should this be `momento = "0.35.0"`?

Reply:
Yes it should! Fixed in [6154297](https://github.com/momentohq/client-sdk-rust/pull/262/commits/6154297fb9a442d3464631af0a072f7136a84967)